clisops/core/subset.py

```diff
@@ -1074,19 +1062,14 @@ def subset_shape(
         try:
             # Extract CF-compliant CRS_WKT from crs variable.
             raster_crs = CRS.from_cf(ds_copy.crs.attrs)
-        except AttributeError as e:
-            # This is guessing that lons are wrapped around at 180+ but without much information, this might not be true
-            if np.min(lon) >= -180 and np.max(lon) <= 180:
-                raster_crs = wgs84
-            elif np.min(lon) >= 0 and np.max(lon) <= 360:
-                wrap_lons = True
-                raster_crs = wgs84_wrapped
```

Comment:
I believe this code might still be necessary for `wrap_lons`, `create_mask` is able to handle the cases where lons are from 0 to 360, but it needs to be told so. 

The  `raster_crs` part is now indeed obsolete.